build_system/src/config.rs

```diff
@@ -469,11 +463,28 @@ impl ConfigInfo {
 }
 
 fn download_gccjit(
-    url: String,
+    commit: &str,
     output_dir: &Path,
     tempfile_name: String,
     with_progress_bar: bool,
 ) -> Result<(), String> {
+    let url = if std::env::consts::OS == "linux" && std::env::consts::ARCH == "x86_64" {
+        format!("https://github.com/rust-lang/gcc/releases/download/master-{}/libgccjit.so", commit)
+    } else {
+        eprintln!(
+            "\
+        Pre-compiled libgccjit.so not available for this os or architecture.\n\
+        Please compile it yourself and update the `config.toml` file\n\
+        to `download-gccjit = false` and set `gcc-path` to the appropriate directory.\
+        "
```

Comment:
Just thinking but at this point we could simply do:

```suggestion
Pre-compiled libgccjit.so not available for this os or architecture.
Please compile it yourself and update the `config.toml` file
to `download-gccjit = false` and set `gcc-path` to the appropriate directory."
```

Reply:
Sure, if that is preferred, done